lib/camunda-platform/features/replace/UnlinkTemplateReplaceProvider.js

```diff
@@ -0,0 +1,57 @@
+import {
+  addPlainElementEntry
+} from '../../../util/UnlinkTemplateUtil';
+
+/**
+ * A replace menu provider that allows to replace elements with
+ * templates applied with the correspondent plain element.
+ */
+export default function UnlinkTemplateReplaceProvider(popupMenu, translate, elementTemplates) {
```

Comment:
Would it hurt to re-use this in the C8 case?

From what I see the C8 `ElementTemplateReplaceProvider` simply builds on top of this component, injecting additional elemnent templates.